src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java

```diff
@@ -96,6 +97,14 @@
      */
     private VertxSliceServer srv;
 
+    static {
+        try (ServerSocket socket = new ServerSocket(0)) {
```

Comment:
@brastak can we move it to `@BeforeAll` or `@BeforeEach` setup method?

Reply:
done. moved to beforeall